WEBROOOOT/style.css

```diff
@@ -0,0 +1,93 @@
+nav {
+  font-style: arial, verdana, sans-serif;
```

Comment:
Nice use of fallback fonts here!  (and in your other declarations as well).

Something for the future: If you plan to use the same fonts across all of the elements of your page, you can put this rule inside your "body" selector - the tags inside the ```<body>``` tag will inherit your font rule so you only have to list it once.